hamcrest/Hamcrest/Arrays/SeriesMatchingOnce.php

```diff
@@ -31,7 +31,7 @@ public function matches($item)
     public function isFinished()
     {
         if (!empty($this->_elementMatchers)) {
-            $nextMatcher = current($this->_elementMatchers);
+            $nextMatcher = reset($this->_elementMatchers);
```

Comment:
We are looking at a difference between PHP and HHVM. I dont thin `reset` or `current` are wrong, it may be more likely that one of the other used functions or even the inputted array had a wrong internal pointer.